pkg/ddevapp/mutagen.go

```diff
@@ -344,8 +344,8 @@ func mutagenSyncSessionExists(app *DdevApp) (bool, error) {
 func (app *DdevApp) MutagenStatus() (status string, shortResult string, mapResult map[string]interface{}, err error) {
 	syncName := MutagenSyncName(app.Name)
 
-	mutagenDataDirectory := os.Getenv("MUTAGEN_DATA_DIRECTORY")
 	fullJSONResult, err := exec.RunHostCommandSeparateStreams(globalconfig.GetMutagenPath(), "sync", "list", "--template", `{{ json (index . 0) }}`, syncName)
+	mutagenDataDirectory := os.Getenv("MUTAGEN_DATA_DIRECTORY")
```

Comment:
Perhaps for these we should be using DDEV logic to find the directory instead of using the environment variable?

Reply:
You are right, I will refactor these.